Jenkinsfile

```diff
@@ -16,18 +16,9 @@ node('lisk-hub') {
 
     stage ('Install npm dependencies') {
       try {
-        sh '''
-        cache_file="$HOME/cache/$( sha1sum package.json |awk '{ print $1 }' ).tar.gz"
-        if [ -f "$cache_file" ]; then
-            tar xf "$cache_file"
-        fi
-        npm install
-        ./node_modules/protractor/bin/webdriver-manager update
```

Comment:
I think the build is failing because this line was removed.

Reply:
Oops, missed that line, sorry.